programs/analyze-suspend/meta.yaml

```diff
@@ -0,0 +1,21 @@
+description: >
```

Comment:
need short_description: >, similar comments for other tests.

Reply:
ok